### ISSUE TITLE
Add logit output option to all models

### ISSUE THREAD

**Issue by yetinam** [seisbench/seisbench#129]
For training, it's usually numerically more stable to train using logits instead of probabilities. The option to output logits should therefore be added. #127 does this for PhaseNet, the others still need to be added. The implementation should be consistent between models.

**Comment by jawooll** [comment]
Looking through #127 - this extension is a useful step and all makes sense, should be a quick fix and I'd be happy to tackle it if needed.

In terms of implementation, just to ensure I'm on the same page, we just define a ```logits``` transform, and add that to the end of all ```forward``` calls where forward can now accept a flag (logits==True | logits ==False), backward compatibility is then enforced as we keep default values for operations set to whatever they were previously (logits=False).

**Comment by yetinam** [comment]
I think there are two ways for implementation:

- The one you suggested, i.e., `def forward(x, logits=False)`.
- Two separate functions, `forward` and `logits`. Here, `forward` internally just calls `logits` and applies the normalization, e.g., softmax.

I don't really have a clear preference. Maybe the first one would be cleaner? What do you think?

**Comment by jawooll** [comment]
Both sound reasonable! I've just had a look through the codebases for TF, PyTorch etc. to see which one would better mirror the de-facto deep learning APIs, with TF calling ```from_logits=False``` etc., in some of the methods  the first case better aligns with that. From my side, I'd agree with you let's go with the first option.

**Comment by yetinam** [comment]
Great, then let's go with the first option. Thanks for taking care of this. I'll then also leave the PR #127 to you. Seems like that needs to be switched from the second to the first option.
